packages/storybook/stories/VideoTile/VideoTile.stories.tsx

```diff
@@ -98,7 +109,7 @@ const VideoTileStory = (args): JSX.Element => {
       showMuteIndicator={args.showMuteIndicator}
       showLabel={args.showLabel}
       isMirrored={args.isMirrored}
-      isMuted={args.isMuted}
+      isMuted={true}
```

Comment:
nit: should this be updated?

Reply:
I'll have to change back the storybook component changes. Thanks.